### ISSUE TITLE
An extremely strange way of transferring energy, is it a bug?

### ISSUE THREAD

**Issue by KrakenTales** [Regalis11/Barotrauma#9883]
### Disclaimers
- [x] I have searched the issue tracker to check if the issue has already been reported.
- [ ]  My issue happened while using mods.

### What happened?
Energy is transferred through components in a strange way.

### Reproduction steps
For convenience, I have placed the entire explanation regarding each situation in the file, the section - assembling items. Add to mods folder.[](url)
[Electricity Issue.zip](https://github.com/Regalis11/Barotrauma/files/9579394/Electricity.Issue.zip)

### Bug prevalence
Happens every time I play

### Version
0.18.15.1

### Which operating system did you encounter this bug on?
Windows

### Relevant error messages and crash reports
No response

**Comment by Regalis11** [comment]
Duplicate of https://github.com/Regalis11/Barotrauma/issues/9603

**Comment by KrakenTales** [comment]
I'm sorry, it looks like I didn't search well, I tried several times to find a possible formulation of the problem, but I did not find this case. Thank you for the feedback!

**Comment by KrakenTales** [comment]
I just read the thread with my case. In this thread of discussion, only 1 case out of 3 that I encountered is presented. As I explained in the notes in the file itself, from the point of view of physics, this is normal. BUT only in those cases that are modeled. If you rearrange the intersecting wires to another contact, the system stops functioning. So as you said previously in #9603

> The way I see it, the power_in pin of a device still only feeds power in to the device, but the two power wires going into it are obviously connected to each other, so there's no reason why power wouldn't be able to move between them.

I can agree with this only in cases where it really works, namely, when I described it above. 
Due to the lack of logic according to the physics of processes or human expectation, I fervently hope that this feature will be corrected, or let all intersecting wires transmit energy further, no matter where they intersect, whether it be energy_in or energy_out, as required by physics and expected in general.
